code/modules/jobs/job_types/legion.dm

```diff
@@ -661,20 +666,47 @@ Weapons		Lever shotgun, Grease gun, Repeater carbines, Revolvers, simple guns al
 	id = /obj/item/card/id/dogtag/legion/prime
 	suit = /obj/item/clothing/suit/armor/f13/legion/vet/explorer
 	head = /obj/item/clothing/head/helmet/f13/legion/vet/explorer
-	glasses = /obj/item/clothing/glasses/sunglasses/big
-	belt = /obj/item/storage/backpack/spearquiver
 	neck = /obj/item/storage/belt/holster
-	suit_store = /obj/item/gun/ballistic/rifle/hunting
 	r_pocket = /obj/item/flashlight
 	l_pocket = /obj/item/binoculars
 	backpack_contents = list(
+		/obj/item/gun/ballistic/revolver/revolver45 = 1,
+		/obj/item/ammo_box/c45rev = 1,
+		/obj/item/reagent_containers/pill/patch/healingpowder = 1,
 		/obj/item/storage/bag/money/small/legenlisted = 1,
-		/obj/item/attachments/scope = 1,
-		/obj/item/melee/onehanded/machete = 1,
+		/obj/item/melee/onehanded/machete/forgedmachete = 1,
 		/obj/item/restraints/handcuffs = 1,
-		/obj/item/ammo_box/a308 = 2,
+		/obj/item/restraints/legcuffs/bola/tactical = 1,
+		/obj/item/storage/survivalkit/medical/legion = 1
+		)
+		
+/datum/outfit/loadout/expscout
+	name = "Scout"
```

Comment:
Remove this loadout that operates the same way as the sniper loadout but with different ammo.